### PR TITLE
Added Biometrics Template

### PR DESCRIPTION
**Pull request by daltonhance**
This creates a template for a submission to Biometrics based off the [journal LaTeX class](http://www.biometrics.tibs.org/latexdocumentclass.htm). There is still some housekeeping in terms of creating a helpfile and exporting the namespace, but I've tested it on my machine.

Apologies for any poor Github etiquette, this is my first external pull request.

### CODE REVIEW

**Comment by daltonhance** [comment]
My fork has been rebased and force pushed. I have added myself to the list of authors, update README.md, and added a test.

File: NEWS [hunk 1]
```diff
@@ -3,6 +3,8 @@ rticles 0.6 (unreleased)
 
 - Added the template for the SAGE Journals (thanks, @oguzhanogreden, #181).
 
+- Added the template for Biometrics (@daltonhance, #170)
```
Comment (yihui) [inline NEWS]: Could you also add an entry in README.md?

File: R/biometrics_article.R [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#' @inheritParams rmarkdown::pdf_document
+#' @param ... Arguments to \code{rmarkdown::pdf_document}
+#' @return R Markdown output format to pass to \code{\link[rmarkdown:render]{render}}
+#' @examples
+#' \dontrun{
+#' rmarkdown::draft("MyArticle.Rmd", template = "biometrics_article", package = "rticles")
+#' }
+#'
+#' @export
+biometrics_article <- function(..., keep_tex = TRUE, citation_package = 'none'){
```
Comment (yihui) [inline R/biometrics_article.R]: Are you sure you don't want to set `citation_package = 'natbib'` as the default?

**Comment by daltonhance** [comment]
I just sent my contributor agreement off. I changed the default citation_package to natbib. I did add a link to the Biometrics journal on the README.md on my fork.

**Pull request merged**